server/core/src/irodsReServer.cpp

```diff
@@ -721,7 +726,7 @@ int main(int argc, char** argv)
         logger::delay_server::error(e.what());
     }
 
-    logger::delay_server::info("Rule execution server exiting ...");
+    logger::delay_server::info("Rule execution server exited normally.");
```

Comment:
this message is never printed when shutting down via the irodsctl script because the control plane sends a SIGKILL after SIGTERM.

POSIX does not allow SIGKILL to be caught.

the control plane should give systems a few seconds to respond to the SIGTERM.

Reply:
this can be handled in a separate PR too.